lib/unison-sqlite/src/Unison/Sqlite/Connection.hs

```diff
@@ -172,9 +172,9 @@ execute conn@(Connection _ _ conn0) sql@(Sql s params) = do
 --
 -- This function does not support parameters, and is mostly useful for executing DDL and migrations.
 executeStatements :: Connection -> Text -> IO ()
-executeStatements conn@(Connection _ _ (Sqlite.Connection database)) sql = do
+executeStatements conn@(Connection _ _ connection) sql = do
   logQuery (Sql sql []) Nothing
-  Direct.Sqlite.exec database sql `catch` \(exception :: Sqlite.SQLError) ->
+  Direct.Sqlite.exec (Sqlite.connectionHandle connection) sql `catch` \(exception :: Sqlite.SQLError) ->
```

Comment:
`sqlite-simple` was upgraded `0.4.18.2 -> 0.4.19.0` and now has multiple entries in the record:

https://hackage.haskell.org/package/sqlite-simple-0.4.18.2/docs/src/Database.SQLite.Simple.Internal.html#connectionHandle
vs 
https://hackage.haskell.org/package/sqlite-simple-0.4.19.0/docs/src/Database.SQLite.Simple.Internal.html#connectionHandle